pro/controllers/users.go

```diff
@@ -1034,6 +1035,8 @@ func getRemoteAccessGatewayConf(w http.ResponseWriter, r *http.Request) {
 		Metadata:          node.Metadata,
 		AllowedEndpoints:  getAllowedRagEndpoints(&node, host),
 		NetworkAddresses:  []string{network.AddressRange, network.AddressRange6},
+		DnsAddress:        node.IngressDNS,
```

Comment:
extclient config already has dns field

Reply:
right. but in that case, we wont be able to show the users the dns field unless they have an extclient